scripts/build-schema.sh

```diff
@@ -7,8 +7,10 @@ version="${VERSION:-@next}"
 
 echo "Version set to $version"
 
-dirs=("application" "prototypeApplication")
-types=("Application" "PrototypeApplication")
+# dirs=("application" "prototypeApplication")
```

Comment:
This looks right to me according to what we chatted about on the dev call yesterday regarding commenting out the stuff we don't want to build yet